packages/babel-generator/src/generators/statements.ts

```diff
@@ -50,22 +50,14 @@ export function IfStatement(this: Printer, node: t.IfStatement) {
 }
 
 // Recursively get the last statement.
-function getLastStatement(
-  statement: Exclude<t.Statement, t.BreakStatement>,
-): t.Statement {
-  if (
-    !isStatement(
-      // @ts-ignore body is not in BreakStatement
-      statement.body,
-    )
-  ) {
+function getLastStatement(statement: t.Statement): t.Statement {
+  // @ts-ignore: If statement.body is empty or not a Node, isStatement will return false
```

Comment:
I have reverted some changes suggested before since `getLastStatement` does accept a BreakStatement, it will return the statement itself as the "last statement".